core/templates/dev/head/pages/admin-page/activities-tab/admin-dev-mode-activities-tab.directive.html

```diff
@@ -92,6 +92,24 @@ <h3>Load dummy new structures data (only admins)</h3>
   </div>
 </md-card>
 
+<md-card class="oppia-page-card oppia-long-text" style="max-width: 700px">
+  <div class="container" style="margin-left: 0;">
+    <h3>Load dummy skill with questions (only admins)</h3>
+    <div class="row">
+      <span class="col-lg-4 col-md-4 col-sm-4">
+        Loads a skill with 15 questions linked with it.
+      </span>
+      <span class="col-lg-2 col-md-2 col-sm-2 ml-auto">
+        <button type="button"
+                class="btn btn-secondary float-right"
+                ng-click="$ctrl.loadNewSkillData()">
+          Load Data
```

Comment:
Generate Dummy Skilll with 15 Questions


Also, one question -- what happens if this button is clicked twice? Should you have a random hash in the skill and question names perhaps? (Maybe the question name includes the name of the skill it was originally linked to.)

Reply:
'Generate Dummy Skilll with 15 Questions' for the button felt a bit too long so I changed it to generate data.

Reply:
I have added a random hash for the name of the skill.